src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
```

Comment:
It's good practice to handle potential edge cases where the CSS string might have empty declarations (e.g., ';;') which would result in empty strings after the `split` operation. Consider filtering out empty strings before the `map` operation.